### PR TITLE
Added mixed cortical layers

### LINKED ISSUE #2084: [NTR] Intermediary layers of neocortex
Cell types in BDS have the following layer information that needs to be added into UBERON:
L2/3
L4/5
L5/6
L6b

For the first 3 I suggest the following: 
Label: cortical layer X/Y etc. 
Definition: A layer of the neocortex that is located between layers X & Y
SubclassOf: layer of neocortex

For L6b: 
Label: cortical layer VIb
Definition: The deepest layer of the neocortex. Neurons in L6b projects to cortical targets and higher-order thalamus . [PMID:32160552]
SubclassOf cortical layer VI
Synonyms: neocortex layer 6b, cortical L6b

Related to: https://github.com/obophenotype/cell-ontology/pull/1270

### LINKED ISSUE #2087: Textual def of layer of neocortex needs expansion
**Uberon term**
UBERON:0002301 [layer of neocortex]

**Suggested revision of textual definition**
Current def: One of the layers of the neocortex.
Issue: This does not allow for multilayer types (eg L2/3) which many cells fall under (see #2084)
Suggested change: One or a combination of multiple layers of the neocortex.

On this note, the x-ref of the current def leads to @cmungall orcid - wonder if I should change this to created_by or dc:contributor or something? (not super sure about conventions in UBERON)

### LINKED ISSUE #2088: not all of cortical layer VI develops from cortical plate
**Uberon term**
UBERON:0005395 (cortical layer VI)

**Suggested revision of logical definition**
Currently it is SubClass of 'develops from' some 'cortical plate' -> this is not technically fully true as while layer VIa does, layer VIb develops from cortical subplate (UBERON:0004035) instead [PMID:32160552]

proposal: remove axiom, and add cortical layer VIa and VIb under cortical layer VI that has develops_from some cortical plate and cortical subplate respectively instead.

### PR DESCRIPTION
**Pull request by shawntanzk**
- Edited def for layer of neocortex to include combination of multiple layer (Fixes #2087)
- added cortical layer II/III, IV/V, V/VI, & VIb (Fixes #2084)
- removed axiom that stats Layer VI develops from cortical plate (Fixes #2088)

Removed all created_by axioms in new terms cause it seems like UBERON terms do not have that.

### CODE REVIEW

**Review by dosumis — CHANGES_REQUESTED** [review]
'layer of neocortex' should be subClassOf 'laminar subdivision of cortex'
add develops_from relationships to specific layers to compensate for deletion of general develops_from relationship on 'layer of neocortex'

**Comment by shawntanzk** [comment]
> 'layer of neocortex' should be subClassOf 'laminar subdivision of cortex' 

Have added is_a 'laminar subdivision of cortex'

> add develops_from relationships to specific layers to compensate for deletion of general develops_from relationship on 'layer of neocortex'

Just checked, each of them has a develops from to more specific terms already so no need for that

**Comment by dosumis** [comment]
> Have added is_a 'laminar subdivision of cortex'

Still not seeing change.

**Comment by shawntanzk** [comment]
> Still not seeing change.

oh sorry - committed it but forgot to push the change - should be there now

**Review by dosumis — APPROVED** [review]

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -101770,7 +101771,6 @@ xref: MA:0000902
 xref: NLXANAT:090812
 xref: PBA:294021896
 is_a: UBERON:0002301 ! layer of neocortex
-relationship: develops_from UBERON:0005343 {source="BRAINSPAN"} ! cortical plate
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:101773]: would it not be appropriate to weaken to has-developmental-contribution-from?

Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:101773]: oh did not know that existed, yes, definitely would be good to add that - will do

**Pull request merged**